src/common/clib-package.c

```diff
@@ -390,6 +390,8 @@ static inline int install_packages(list_t *list, const char *dir, int verbose) {
   loop_cleanup:
     if (slug)
       free(slug);
+    if (error && pkg)
+      clib_package_free(pkg);
```

Comment:
We should add this to the existing `if (error)` block:

```c
if (error) {
  if (pkg) {
    clib_package_free(pkg);
    pkg = NULL;
  }
}

Reply:
fix